src/main/java/com/bugsnag/android/gradle/BugsnagVariantOutputUtils.java

```diff
@@ -1,7 +1,8 @@
 package com.bugsnag.android.gradle;
 
-import com.android.build.gradle.api.BaseVariant;
-import com.android.build.gradle.api.BaseVariantOutput;
+import com.android.build.gradle.api.ApkVariant;
+import com.android.build.gradle.api.ApkVariant;
```

Comment:
These two lines are the same

```suggestion
import com.android.build.gradle.api.ApkVariant;
```